### DISCUSSION TITLE
When "tuh_hid_set_report" calls return false, how can I recover?

### DISCUSSION THREAD

**Discussion by morio** [hathach/tinyusb#1805] [Q&A]
Hi, I'm using `tuh_hid_set_report()` to change LEDs for NumLock, CapsLock, and ScrollLock but after a while the calls start to return false.
Is there a way to recover after `tuh_hid_set_report` starts failing?

On a side note, how do I enable TinyUSB Logging with a custom RP2040 board that is using a different UART TX pin?

Thanks,
-Morio S.

**Comment by hathach** [comment]
an control transfer return false if there is already an on-going control transfer. You can enable log by chaing board.h or change pin uart depending on how you compile your code.
